Dockerfile

```diff
@@ -69,6 +69,23 @@ RUN xx-apt-get install -y binutils gcc libc6-dev libgcc-10-dev libsecret-1-dev p
 FROM base AS test
 ARG DEBIAN_FRONTEND
 RUN xx-apt-get install -y dbus-x11 gnome-keyring gpg-agent gpgconf libsecret-1-dev pass
+RUN <<EOF
+curl \
+    -o /usr/share/keyrings/gopass-archive-keyring.gpg \
+    https://packages.gopass.pw/repos/gopass/gopass-archive-keyring.gpg 
+
+    cat <<- DEBSOURCE > /etc/apt/sources.list.d/gopass.sources
+        Types: deb
+        URIs: https://packages.gopass.pw/repos/gopass
+        Suites: stable
+        Architectures: all amd64 arm64 armhf
+        Components: main
+        Signed-By: /usr/share/keyrings/gopass-archive-keyring.gpg
+    DEBSOURCE
+
+    xx-apt-get update
+    xx-apt install gopass gopass-archive-keyring
+EOF
```

Comment:
I would prefer to pin gopass version and would also be easier to install gopass from GitHub Releases: https://github.com/gopasspw/gopass/releases

Something like:

```dockerfile
ARG GOPASS_VERSION=1.15.5
ARG TARGETARCH
ARG TARGETVARIANT
RUN wget -qO- "https://github.com/gopasspw/gopass/releases/download/v${GOPASS_VERSION}/gopass-${GOPASS_VERSION-linux-${TARGETARCH}${TARGETVARIANT}.tar.gz" | tar -zxvf - gopass
```

Reply:
I guess we could even

```dockerfile
RUN --mount=type=bind,target=. \
    --mount=type=cache,target=/root/.cache \
    --mount=type=cache,target=/go/pkg/mod \
   go install github.com/gopasspw/gopass@${GOPASS_VERSION}
```

but not sure if we want to install from source

Reply:
The reason I went for installing via `apt` is because that's how `pass` is currently handled, and the version is not pinned. That said, I'd far prefer pinning a specific version.

Personally, I'd build from source; I'll send up a new revision for doing that.

Reply:
On second thought, building from source might not be ideal due to the increased time it brings to the build. Any further thoughts on a preference for either approach, @crazy-max and @thaJeztah?

Reply:
Some general thoughts:

- Building from source might be more "secure" than downloading a release from GitHub that isn't signed
- If we pin a version, that's good for stability, but puts the onus on this maintainers to bump the version of the downstream consumer in order to validate the credential helper with new versions of `gopass`. As I mentioned above, this is not how `pass` is explicitly handled; instead, `pass` can be (silently) updated due to `apt-get update` calls prior to `apt-get install ... pass`, which causes the image build to be non-hermetic (although there are other reasons it is non-hermetic as well).